.github/workflows/drivers.yml

```
@@ -21,8 +21,15 @@ on:
       - "**/frontend/**.unit.*"
 
 jobs:
+  draft-check:
+    if: github.event.pull_request.draft == false
+    runs-on: ubuntu-20.04
+    timeout-minutes: 5
+    steps:
+      - run: echo "PR is not in a draft mode. Continue with the rest of the jobs."
 
   be-tests-druid-ee:
+    needs: draft-check
```

@ariya alternatively, we can use the conditional directly in each of the jobs:

```suggestion
    if: github.event.pull_request.draft == false
```

What do you prefer?

---

The latter, since we already use it in e.g. `e2e-tests.yml`.